### PR TITLE
Replace manual language client library with GAPIC-only surface

### PR DESCRIPTION
**Pull request by geigerj**
Wholesale replacement of `google-cloud-language/` directory with autogen.

Note: includes manual edits to link to migration guide for README,
post-install message

### CODE REVIEW

**Comment by blowmage** [comment]
FYI, this PR is conflicting and needs to be updated.

**Comment by quartzmo** [comment]
@geigerj Thanks for the updates. I've pulled the latest and am reviewing now.

**Comment by quartzmo** [comment]
@geigerj When running `rake ci:a` in `google-cloud-language`, I get this error:

```sh
The SMOKE_TEST_PROJECT environment variable must be set. e.g SMOKE_TEST_PROJECT=test123 rake smoke_test
```

Should I permanently add this variable to my local environment, and does it need to be added to CI environments as well?

**Comment by geigerj** [comment]
@quartzmo Thanks for pointing that out. No, we should be able to reuse the existing Google Cloud env variables. I would like to think about this a bit more about how we can treat this properly in the code generator, since we don't want to tie the generated Rakefile specifically to Google Cloud. In this case, since there are no smoke tests being checked in yet, I'm going to remove this check in the Rakefile manually.

**Comment by blowmage** [comment]
FWIW, this is how this was solved in the Monitoring gem, using the existing ENV variables and `smoke_test` task:

https://github.com/GoogleCloudPlatform/google-cloud-ruby/commit/cae8928c94a89476d2883a4a50daa7acd73173ea

**Comment by blowmage** [comment]
GitHub is still showing this PR has merge conflicts.

**Comment by geigerj** [comment]
@blowmage Thanks, the monitoring solution seems like a good interim tack. I'm not seeing merge conflicts in the GitHub UI, but I'll merge in master in case there's some latency...

**Comment by geigerj** [comment]
@blowmage Actually, I'm seeing "Already up to date" when I try to merge in master. I think this PR should be OK.

File: google-cloud-language/README.md [hunk 1]
```diff
@@ -1,72 +1,48 @@
...
+- [Product Documentation][]
 
-The Cloud Natural Language API currently supports English, Spanish, and Japanese for sentiment analysis, entity analysis, and syntax analysis.
-
-- [google-cloud-language API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-language/latest)
-- [google-cloud-language on RubyGems](https://rubygems.org/gems/google-cloud-language)
-- [Google Cloud Natural Language API documentation](https://cloud.google.com/natural-language/docs)
+*This release, 0.28.0, introduces breaking changes relative to the previous
+release, 0.27.1. For more details and instructions to migrate your code, please
+visit the [migration guide](https://cloud.google.com/natural-language/docs/ruby-client-migration).*
```
Comment (quartzmo) [inline google-cloud-language/README.md]: I'm getting a 404 for the migration guide URL.

Comment (geigerj) [reply google-cloud-language/README.md]: Yes, that's intentional. It cannot go live until samples are published, and samples cannot be published until the gem is published... We'll coordinate (hopefully tomorrow) with @frankyn to ensure there's a minimal amount of time during which this link is broken.

File: google-cloud-language/Rakefile [hunk 2]
```diff
@@ -61,10 +66,32 @@ end
...
+  # Required for smoke tests
+  ENV["SMOKE_TEST_PROJECT"] = project
+
   Rake::Task["smoke_test"].invoke
 end
 
+
+
 namespace :acceptance do
   task :run do
```
Comment (blowmage) [inline google-cloud-language/Rakefile]: This task seems out of place. Is the intention that the `acceptance` task is supposed to call `acceptance:run`?

**Comment by blowmage** [comment]
@geigerj Strange. GitHub still shows "This branch cannot be rebased due to conflicts" and I get conflicts when attempting to rebase this branch on master in my local repo. ¯\\\_(ツ)\_/¯

I'll merge as soon as the build is green.

**Comment by quartzmo** [comment]
I created a new gemset, built and installed the package, and tested the README example in IRB. Everything worked great.

File: google-cloud-language/README.md (hunk 1 above)
Comment (frankyn) [reply google-cloud-language/README.md]: That's goal! Minimize the delay for that.

File: google-cloud-language/Rakefile (hunk 2 above)
Comment (geigerj) [reply google-cloud-language/Rakefile]: Yes, that's right, this is a code generator bug. Both these tasks are currently no-ops right now, but will fix this in the next refresh. Tracking as https://github.com/googleapis/toolkit/issues/1688.

**Comment by geigerj** [comment]
@blowmage Weird, I see: occhd77a8vw

If it's blocking you from merging, @quartzmo, could you do the merge? I can also merge myself if there's some GitHub bug blocking both of you...

**Pull request merged**